config.py

```diff
@@ -12,41 +13,74 @@
 
 
 class FilterType(str, Enum):
+    """What to do if the opponent declines our challenge."""
+
     NONE = "none"
+    """Will still challenge the opponent."""
     COARSE = "coarse"
+    """Won't challenge the opponent for some time."""
     FINE = "fine"
+    """Won't challenge the opponent to a game of the same mode, speed, and variant for some time."""
```

Comment:
Replace "for some time" with "again based on the reason for the opponent declining the challenge."